js/lightbeam.js

```diff
@@ -189,7 +189,7 @@ const lightbeam = {
     let links = [];
     for (const website in this.websites) {
       const site = this.websites[website];
-      if (site.firstParty) {
+      if (site.thirdParties) {
```

Comment:
This is saying: We are actually just interested to know what the new links are for a site, which could be both a first and a third party, so let's check for that instead.